test/test.js

```diff
@@ -0,0 +1,47 @@
+const chai = require('chai');
```

Comment:
You'd like to have your tests using some stub data.
Currently you're using a local array. But what if you will use a DB one day. You won't like your tests to change things in the DB.

Reply:
Super shallow tests